src/Executables/Auth/PacketHandlers/LoginRequestHandler.cs

```diff
@@ -86,11 +118,23 @@ public async Task ExecuteAsync(AuthPacketContext<LoginRequest> ctx, Cancellation
         // Set expiration on token
         await _cacheManager.Expire("token:" + authToken, 30);
 
+        // Relate the account ID to the token
+        await _cacheManager.Set("account:token:" + account.Id, authToken);
```

Comment:
Bad baseline but I prefer string interpolation `$"account:token:{account.Id}"`. Not your fault - you are adapting the given style. But maybe something we can cleanup when we are at it?